Tests/TABResourceLoaderTests/Protocols/NetworkJSONResourceTypeTests.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  NetworkJSONResourceTypeTests.swift
+//  TABResourceLoader
+//
+//  Created by John Sanderson on 30/01/2018.
+//  Copyright © 2018 Luciano Marisi. All rights reserved.
```

Comment:
Maybe use the MIT license and the TAB copyright here.

Reply:
Updated